### PR TITLE
fix gpu-onnx infer

### PR DESCRIPTION
**Pull request by LRY1994**
fix eos sos to depending on dict or config

### CODE REVIEW

**Comment by xingchensong** [comment]
请根据这里的提示 https://github.com/wenet-e2e/wenet/actions/runs/9762251705/job/26970942324#step:5:1       修复lint错误

**Comment by xingchensong** [comment]
UnitTest 错误看起来和本PR无关，请 @Slyne 审核下，没啥问题可以直接合并 ：）

File: runtime/gpu/model_repo_stateful/wenet/1/wenet_onnx_model.py
Comment (Slyne) [inline runtime/gpu/model_repo_stateful/wenet/1/wenet_onnx_model.py:1]: Hi, `cur_mask_len` 这个改动是必须的吗？

File: wenet/bin/recognize_onnx_gpu.py
Comment (Slyne) [inline wenet/bin/recognize_onnx_gpu.py:1]: Can you remove the blank lines? Or just use one blank line instead of two lines?  Thanks!

**Review by Slyne — COMMENTED** [review]
Hi, thanks for the PR to make it flexible to read blankid, eos, sos id from config file.
Codes look good to me. 
Please refer to the comments about formatting.

Appreciated!

File: runtime/gpu/model_repo_stateful/wenet/1/wenet_onnx_model.py
Comment (LRY1994) [reply runtime/gpu/model_repo_stateful/wenet/1/wenet_onnx_model.py:1]: 源代码里面有hist_enc为None的情况，len(None)会报错，测试的时候确实也有出现hist_enc为None的情况。
```
if hist_enc is None:
  cur_enc = cur_encoder_out[idx]
.....
cur_mask_len = int(len(hist_enc) + seq_lens[idx])
```

Comment (Slyne) [reply runtime/gpu/model_repo_stateful/wenet/1/wenet_onnx_model.py:1]: Thanks!   Didn't realize we changed the initial history cache.  FYI, @yuekaizhang   Didn't we find this issue before?

**Review by Slyne — APPROVED** [review]
Thanks!

**Pull request merged**